python/spacewalk/satellite_tools/reposync.py

```diff
@@ -1128,7 +1138,9 @@ def import_groups(self, plug):
     def import_modules(self, plug):
         modulesfile = plug.get_modules()
         if modulesfile:
-            self.copy_metadata_file(plug, modulesfile, "modules", relative_modules_dir)
+            return self.copy_metadata_file(
+                plug, modulesfile, "modules", relative_modules_dir
+            )
```

Comment:
Why was this changed?

Reply:
Are you running black from the container? `reposync.py` was already reformatted with black, I wonder why yours has a different line length